.circleci/config.yml

```diff
@@ -129,7 +129,7 @@ jobs:
           no_output_timeout: 30m
       - run:
           name: Verify Bundle Size
-          command: npx bundlesize2 --enable-github-checks
+          command: ./node_modules/.bin/bundlesize --enable-github-checks
```

Comment:
threw me for a loop there... did not realize that `bundlesize2` is using a binary called `bundlesize`... cause that's not confusing lol